accounting/localizations/argentina.rst

```diff
@@ -0,0 +1,280 @@
+
+2.1.2 AFIP Responsibility Type
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+In Argentina the document type associated with customers and vendors transactions is defined based on the AFIP Responsibility type, this field should be defined in the partner form:
+
+.. image:: media/argentina05.png
+   :align: center
+
+
+2.2 Taxes
++++++++++
+
+As part of the localization module,  the taxes are created automatically with their related financial account and configuration.
+
+.. image:: media/argentina06.png
+   :align: center
+
+2.2.2 Taxes Classification
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#. **Type**:
+-  VAT. Is the regular VAT and it can have several percentages.
+-  Perception. Advance payment of a tax that is applied on Invoices.
+-  Retention.  Advance payment of a tax that is applied on payments
+-  Otros.
+
+
+#. **Tax Category**:
+- Regular VAT
+- Profits
+- Gross Income
+- Otros
+
+#. **Application**:
+- National
+- Provincial
+- Municipal
+- Impuestos Internos
+- Otros
+
+
+2.2.3 Special Taxes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Some argentine taxes are not commonly used  for all companies, these type of taxes are included as Inactive by default, it's important that before creating a new tax you confirm if they are not already included in the Inactive taxes:
+
+.. image:: media/argentina07.png
+   :align: center
+
+
+2.3 Document Types
+++++++++++++
+In some Latinamerica countries, including Argentina, some accounting transactions like invoices and vendor bills are classified by  document types defined by the government fiscal authorities (In Argentina case: AFIP).
+
+The document type is an essential information that needs to be displayed in the printed reports and that needs to be easily identified, within the set of invoices as well of account moves.
+
+Each document type can have a unique sequence per journal where it is assigned. As part of the localization, the Document Type include the country on which the document is applicable and the data is created automatically when the localization module is installed.
+
+The information required for the document types is included by default so the user doesn't need to fill anything on this view:
+
+.. image:: media/argentina08.png
+   :align: center
+
+Note:  There are several document types that inactive by default but they can be activated if needed.
+
+
+2.3.1 Letters
+^^^^^^^^^^^^^^^^^^^^
+For Argentina, the document types include a letter that helps that indicates the transaction/operation, example:
+
+When an invoice is related to a B2B transaction, a document type "A" must be used.
+When an invoice is related to a B2C transaction, a document type "B" must be used.
+When an invoice is related to exportation transaction, a document type "E" must be used.
+The documents included in the localization have the proper letter associated, the user doesn't need to configure anything additional.
+
+.. image:: media/argentina09.png
+   :align: center
+
+Use on Invoices
+2.3.1 Use on Invoices
```

Comment:
I believe new line is needed here so that it is rendered correclty
Also, the number should be 2.3.2 (instead of .1)